ts/receiver/message.d.ts

```diff
@@ -0,0 +1,56 @@
+enum MessageModelType {
```

Comment:
same here, this should be in
`js/models/message.d.ts`

Reply:
moved